src/cli.js

```diff
@@ -167,120 +166,97 @@ function checkContributors(argv) {
+    )
+  })
+
+  issueCreators.forEach(usr => {
+    const contributor = {
+      login: usr.login,
+      contributions: [],
+    }
+
+    usr.labels.forEach(lbl => {
+      const guessedCategory = learner.classify(lbl).find(c => c && c !== 'null')
 
-      repoContributors.commitAuthors.forEach(usr => {
-        // const contributor = {
-        //   login: usr.login,
-        //   contributions: [],
-        // }
-        // console.log('commit auth:', usr)
-        const existingContributor = contributorsToAdd.filter(
-          ctrb => ctrb.login === usr.login,
+      if (!guessedCategory) {
+        console.warn(
+          `Oops, I couldn't find any category for the "${lbl}" label`,
         )
-        if (existingContributor.length) {
-          //there's no label or commit message info so use only code for now
-          if (!existingContributor[0].contributions.includes('code')) {
-            existingContributor[0].contributions.push('code')
-          }
-        } else
-          contributorsToAdd.push({login: usr.login, contributions: ['code']})
-      })
 
-      // console.log('contributorsToAdd=', contributorsToAdd)
-      contributorsToAdd.forEach(contributor => {
+        return
+      }
+
+      if (!contributor.contributions.includes(guessedCategory)) {
+        contributor.contributions.push(guessedCategory)
+
         console.log(
-          `Adding ${chalk.blue(contributor.login)} for ${chalk.underline(
-            contributor.contributions.join('/'),
+          `Adding ${chalk.blue(usr.login)} for ${chalk.underline(
+            guessedCategory,
           )}`,
         )
-        args._ = ['', contributor.login, contributor.contributions.join(',')]
-        // if (contributor.contributions.length) addContribution(args)
-        // else console.log('Skipping', contributor.login)
-      })
-    },
-    err => console.error('fetch error:', err),
-  )
+      }
+    })
+
+    const existingContributor = contributorsToAdd.find(
+      c => c.login === usr.login,
+    )
+
+    if (existingContributor) {
+      existingContributor.contributions.push(...contributor.contributions)
+    } else {
+      contributorsToAdd.push(contributor)
+    }
+  })
+
+  commitAuthors.forEach(usr => {
+    const existingContributor = contributorsToAdd.find(
+      c => c.login === usr.login,
+    )
+
+    if (existingContributor) {
+      // There's no label or commit message info so use only code for now
+      if (!existingContributor.contributions.includes('code')) {
+        existingContributor.contributions.push('code')
+      }
+    } else {
+      contributorsToAdd.push({login: usr.login, contributions: ['code']})
+    }
+  })
+
+  // TODO: Roll onto other contribution categories following https://www.draw.io/#G1uL9saIuZl3rj8sOo9xsLOPByAe28qhwa
+
+  for (const contributor of contributorsToAdd) {
+    if (!contributor.contributions.length) {
+      console.log('Skipping', contributor.login)
+
+      return
```

Comment:
This should be `continue` right or did you actually wanted it to get out of the `fetchContributors` function?

Reply:
Oh good catch!